trade-finance-manager-ui/component-tests/case/cancellation/effective-from-date.component-test.ts

```diff
@@ -31,17 +31,18 @@ describe(page, () => {
     wrapper.expectLink('[data-cy="cancel-link"]').toLinkTo(`/case/${dealId}/cancellation/cancel`, 'Cancel');
   });
 
-  it('should render back link button linking to the deal bank request date page', () => {
+  it('should render back link button with the passed in URL', () => {
     // Arrange
     const dealId = 'dealId';
-    const effectiveFromDateViewModel: EffectiveFromDateViewModel = { ...aEffectiveFromDateViewModel(), dealId };
+    const backUrl = 'testUrl';
+    const effectiveFromDateViewModel: EffectiveFromDateViewModel = { ...aEffectiveFromDateViewModel(), dealId, backUrl };
```

Comment:
FWIW I've been using `previousPage` - happy to stick with this if both are used but want to try keep consistent :)